ontoconv/pipelines.py

```diff
@@ -217,22 +218,27 @@ def generate_ontoflow_pipeline(
 
         ```python
         {'sinks': [
-           {'iri': 'ss3:AbaqusConfiguration', 'resourcetype': 'ss3:AbaqusSimulation'},
-           {'iri': 'ss3:AluminiumMaterialCard', 'resourcetype': 'ss3:AbaqusSimulation'},
-           {'iri': 'ss3:ConcreteMaterialCard', 'resourcetype': 'ss3:AbaqusSimulation'}],
+           {'iri': 'ss3:AbaqusConfiguration',
+            'resourcetype': 'ss3:AbaqusSimulation'},
+           {'iri': 'ss3:AluminiumMaterialCard',
+            'resourcetype': 'ss3:AbaqusSimulation'},
+           {'iri': 'ss3:ConcreteMaterialCard',
+            'resourcetype': 'ss3:AbaqusSimulation'}],
          'sources': [
            {'iri': 'ss3kb:abaqus_config1', 'resourcetype': 'dataset'},
-           {'iri': 'ss3:AluminiumMaterialCard', 'resourcetype': 'ss3:MaterialCardGenerator'},
-           {'iri': 'ss3kb:abaqus_materialcard_concrete1', 'resourcetype': 'dataset'}]}
+           {'iri': 'ss3:AluminiumMaterialCard',
+            'resourcetype': 'ss3:MaterialCardGenerator'},
+           {'iri': 'ss3kb:abaqus_materialcard_concrete1',
+            'resourcetype': 'dataset'}]}
         ```
     """
-    names = []
+    names = {"input": [], "output": []}
     strategies = []
 
     lst = [("output", s) for s in resources.get("sources", [])]
```

Comment:
Instead of reformatting the `resources` from your OntoFlow output parser to `lst`, it might be possible to do this reformatting already in the OntoFlow output parser.

Reply:
sure, I have jsut based this PR on yours, and the list was your suggestions :)